### ISSUE TITLE
Add support to Thunderbird, and Firefox for Android, Icemonkey

### ISSUE THREAD

**Issue by davidhedlund** [mozilla/addon-compatibility-reporter#77]
Add-on Compatibility Reporter only support Firefox which is useful to see which installed add-ons that are compatible with _multiprocess_.

I need to test extensions for
* Thunderbird
* Icemonkey
* Firefox for Android

Can you please add support for them?

**Comment by davidhedlund** [comment]
[mylabel:important](https://github.com/search?utf8=%E2%9C%93&q=commenter%3Adavidhedlund+%22mylabel%3Aimportant%22+in%3Acomments&type=Issues&ref=searchresults)

**Comment by wagnerand** [comment]
Which of these products use e10s?

Our resources are very limited and the path forward is WebExtensions only (which are already e10s compatible by default), but I think I would accept a patch for this.

**Comment by davidhedlund** [comment]
Only 
https://addons.mozilla.org/en-US/firefox/collections/DavidHedlund/electrolysis-security/ 
use e10s.

**Comment by wagnerand** [comment]
You were asking to add support for other products. Which of these products use/support/offer e10s?

**Comment by davidhedlund** [comment]
No, I just used the directory.fsf.org link as a reference to my long 
work as a add-on repo maintainer, so you can take me seriously.

**Comment by wagnerand** [comment]
Alright, what exactly is your ask here?

**Comment by davidhedlund** [comment]
@wagnerand I'm sorry, I thought I was responding to another conversation. Can you please delete your replies so we can start all over?

This issue is real.

**Comment by davidhedlund** [comment]
> Which of these products use e10s?

* Thunderbird [FTP](https://ftp.mozilla.org/pub/thunderbird/releases/) says 45.7.1: Not Electrolysis yet.
* SeaMonkey -- I don't understand the version scheme. Will it even be Electrolysis compatible?
* Firefox for Android

> 
> Our resources are very limited and the path forward is WebExtensions only (which are already e10s compatible by default), but I think I would accept a patch for this.

**Comment by davidhedlund** [comment]
@wagnerand I updated the Issue. Can you please delete your comments?

**Comment by davidhedlund** [comment]
@wagnerand Lol, the reasons I got confused earlier was because we emailed in another conversation at the same time. So I saw your name again.

**Comment by davidhedlund** [comment]
@wagnerand Can we finish this conversation?

**Comment by wagnerand** [comment]
> Our resources are very limited and the path forward is WebExtensions only (which are already e10s compatible by default), but I think I would accept a patch for this.

I think this is still correct.

**Comment by davidhedlund** [comment]
@wagnerand Yes, but this make it impossible to find out if a Firefox-Android add-on is compatible with Electrolysis. As long as you don't state it on the add-on AMO page.

**Comment by davidhedlund** [comment]
Define multiprocess compatibility on add-on pages on AMO

**Comment by davidhedlund** [comment]
https://github.com/mozilla/addons-server/issues/4975#issuecomment-288103207
